roles/config/tasks/activation_key.yml

```diff
@@ -0,0 +1,78 @@
+---
+- name: "Create generic_key activation key"
+  theforeman.foreman.katello_activation_key:
+    username: "{{ lookup('env','FOREMAN_USER') }}"
```

Comment:
These lookups are a fine default value, but should be able to be parameterized in Ansible. They should live under `defaults/main.yml` and be named something like this

```yaml
satellite_config_foreman_user: "{{ lookup('env', 'FOREMAN_USER') }}"
```

Reply:
I'll only mention it here, but the same pattern should apply throughout the roles.